src/xrpld/app/tx/detail/AMMClawback.cpp

```diff
@@ -42,9 +42,6 @@ AMMClawback::preflight(PreflightContext const& ctx)
     if (auto const ret = preflight1(ctx); !isTesSuccess(ret))
         return ret;  // LCOV_EXCL_LINE
 
-    if (ctx.tx.getFlags() & tfAMMClawbackMask)
```

Comment:
I think this check should remain here. The flags are usually the first ones to be checked after `preflight1` check.